packages/ui-extensions/docs/surfaces/customer-account/staticPages/upgrading-to-2025-10-rc.doc.ts

```diff
@@ -33,12 +33,12 @@ We advise against migrating your production customer account UI extension to Pol
 |   **Legacy Component**   |   **Polaris Web Component**   |   **Migration Notes**   |
 | :----------------------: | :-------------------------------: | :---------------------: |
 |   \`Avatar\`                |   [Avatar](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/avatar)   |   Available today.   |
-|   \`Card\`                  |  [Section](/api/checkout-ui-extensions/2025-10-rc/polaris-web-components/section)                                             |   Available today.                     |
+|   \`Card\`                  |  [Section](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/section)   |   Available today.   |
 |   \`CustomerAccountAction\`  |  [CustomerAccountAction](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/customeraccountaction)   |   Available today.       |
 |   \`ImageGroup\`            |  [ImageGroup](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/imagegroup)                         |   Available today.       |
 |   \`Menu\`                  |  [Menu](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/menu)    |   Available today.   |
 |   \`Page\`                  |  [Page](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/page)    |   Available today. |
-|   \`ResourceItem\`          |                                        |   Coming soon.  |
+|   \`ResourceItem\`          |  [Section](/api/customer-account-ui-extensions/2025-10-rc/polaris-web-components/section)   |   Available today.   |
```

Comment:
Since we want people to use `Section` to replace `ResourceItem`, I figured this made sense. Thoughts?

Reply:
Maybe instead of `Available today` we should say `Use Section` and link to it?

Reply:
Like this? Or also keeping Section in the "Polaris Web Component" column?

<img width="613" height="394" alt="Screenshot 2025-08-28 at 8 54 01 AM" src="https://github.com/user-attachments/assets/2d931b60-c613-42f0-8690-49a077d2e25e" />